storeshare_api/spec/factories/listings.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :listing do
+    
```

Comment:
just curious--is this part of setting up tests for the controllers?

Reply:
not right now, but I intend to add some controller tests for `listings` later and will prob use a listing factory